src/status_im/contexts/wallet/create_account/new_keypair/check_your_backup/view.cljs

```diff
@@ -62,20 +61,26 @@
 
 (defn- view-internal
   []
-  (let [random-indices  (random-selection)
-        quiz-index      (reagent/atom 0)
-        incorrect-count (reagent/atom 0)
-        show-error?     (reagent/atom false)]
+  (let [random-indices                        (random-selection)
+        quiz-index                            (reagent/atom 0)
+        incorrect-count                       (reagent/atom 0)
+        show-error?                           (reagent/atom false)
+        {:keys [secret-phrase random-phrase]} (rf/sub [:wallet/ui])]
     (fn []
-      (let [current-word-index        (get random-indices (min @quiz-index (dec questions-count)))
-            current-word              (get temp/secret-phrase current-word-index)
-            [options-r-0 options-r-1] (random-words-with-string temp/random-words current-word)
+      (let [current-word-index        (get random-indices
+                                           (min @quiz-index (dec questions-count)))
+            current-word              (get secret-phrase current-word-index)
+            [options-r-0 options-r-1] (random-words-with-string random-phrase current-word)
```

Comment:
this naming is a bit confusing, I think we should just be verbose here. 
e.g `options-row-0` `options-row-1`